docs/parsing.rst

```diff
@@ -239,3 +239,21 @@ Stricter parsing can be accomplished by passing a different grammar object
   Traceback (most recent call last):
     ...
   pvl.lexer.LexerError: (LexerError(...), 'Expecting an Aggregation Block, an Assignment Statement, or an End Statement, but found "#" : line 3 column 1 (char 67)')
+
+----------
+From a URL
+----------
+
+The :func:`pvl.loadu()` function returns a Python object (typically a
+:class:`pvl.PVLModule` object which is :class:`dict`-like) based on
+parsing the PVL text in the data returned from a URL.
+
+This is very similar to parsing PVL text from a file, but you use
+:func:`pvl.loadu()` instead::
+
+ >>> import pvl
+ >>> url = 'https://raw.githubusercontent.com/planetarypy/pvl/master/tests/data/pds3/simple_image_1.lbl'
```

Comment:
maybe replace with a url to a pds label that we know is correctly formatted

Reply:
Yeah, that would probably be better for the example here.

Reply:
Put in a HiRISE URL.